docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.9"
```

Comment:
This is used to spin up a local elastic search. It is useful for integration test(s)